### ISSUE TITLE
[BUG]  No proper colour contrast in Dark Mode

### ISSUE THREAD

**Issue by nsk6704** [JAYESHBATRA/Virtuo-Learn#211]
### Description

The Website does not have a smooth colour contrast and impacts visibility of certain components.

### Screenshots

![Screenshot_20240511_105701_Chrome](https://github.com/JAYESHBATRA/Virtuo-Learn/assets/117816770/3b412585-d82c-4362-9d7c-6f55b171d1b7)
![Screenshot_20240511_105652_Chrome](https://github.com/JAYESHBATRA/Virtuo-Learn/assets/117816770/8ecebf3a-a170-4952-8222-3a5bacb31ade)

### Checklist

- [X] I have starred the repository
- [X] I have checked the existing issues
- [X] I have read the discussion tab thoroughly and got the project idea
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSOC'24 contributor

**Comment by nsk6704** [comment]
I kindly request you to assign this issue to me, an aspiring web developer certified by freeCodeCamp in Web Design and have also interned at a company named Codsoft.

Here is my LinkedIn Profile for more information!

https://www.linkedin.com/in/saketh-kashyap-nagendra/
